help/email-notifications.md

```diff
@@ -68,10 +52,34 @@ Zulip Cloud organizations, or in multiple organizations on the same Zulip server
 
 {end_tabs}
 
+### Configure delay for message notification emails
```

Comment:
Test failure looks to be a couple links that need to be updated for this having a new anchor.